configs/projects/puppet-agent.rb

```diff
@@ -96,6 +96,8 @@
 
   proj.setting(:ruby_version, "2.4.3")
   proj.setting(:gem_home, File.join(proj.libdir, "ruby", "gems", "2.4.0"))
+  # Directory for gems shared by puppet and puppetserver
+  proj.setting(:puppet_vendor_dir, File.join(proj.libdir, "ruby", "vendor_gems"))
```

Comment:
Should this maybe be `puppet_gem_vendor_dir` so it's more obvious this is related to Gem stuff?

Reply:
Agreed. I just started with the name you'd chosen in your previous PR.